src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Ahh so it's not needed as if you want to build Swagger docs you could just recompile it before